uuid.js

```diff
@@ -59,6 +59,16 @@
     _hexToByte[_byteToHex[i]] = i;
   }
 
+  // Force _hexToByte to be changed to fast-property object in V8
+  (function toFastProperties(obj) {
+    /*jshint -W027*/
+    function f() {}
+    f.prototype = obj;
+    return f;
+    eval(obj);
```

Comment:
Does this serve a purpose?  Isn't this ...

  1. unreachable
  2. a noop (eval just returns `obj`, right?)